test/refunder.ts

```diff
@@ -278,22 +309,29 @@ describe("Refunder", function() {
 		});
 
 		it('Should NOT be refunded, gas price is too expensive', async () => {
+
+			const userAddress = addr1.address;
+
 			const funcIdAsBytes = generateFuncIdAsBytes('setGreeting(string)');
 			
 			let temp = await refunder.setMaxGasPrice('2000000000000')
 			await temp.wait();
 			
-			let res = await refunder.whitelistRefundable(greeter.address, funcIdAsBytes, true);
+			let res = await refunder.updateRefundable(greeter.address, funcIdAsBytes, true);
 			await res.wait();
 
 			const text = 'Hello, Tester!';
 			const hexString = ethers.utils.formatBytes32String(text);
 			const args = ethers.utils.arrayify(hexString);
+			
+			const balanceBefore = await ethers.provider.getBalance(userAddress);
 
 			await expect(refunder.connect(addr1).relayAndRefund(greeter.address, funcIdAsBytes, args, {
 				gasPrice: '2000000000001'
 			})).to.be.revertedWith(TOO_EXPENSIVE_GAS_PRICE);
 
+			const balanceAfter = await ethers.provider.getBalance(userAddress);
+			expect(balanceAfter.lt(balanceBefore), 'User was refunded').to.be.ok;
```

Comment:
This is not the way we should check whether he was refunded or not. We need to check that his balance is:
`balance = balanceBefore - txCost`